Core/Core/View/Base/WebUnitViewModel.swift

```diff
@@ -8,17 +8,17 @@
 import Foundation
 import SwiftUI
 
-public class WebUnitViewModel: ObservableObject {
+public class WebUnitViewModel: ObservableObject, WebviewCookiesUpdateProtocol {
     
-    let authInteractor: AuthInteractorProtocol
+    public let authInteractor: AuthInteractorProtocol
     let config: ConfigProtocol
     
-    @Published var updatingCookies: Bool = false
-    @Published var cookiesReady: Bool = false
-    @Published var showError: Bool = false
+    @Published public var updatingCookies: Bool = false
+    @Published public var cookiesReady: Bool = false
+    @Published public var showError: Bool = false
     private var retryCount = 1
```

Comment:
property `retryCount` was defined and never used same inside `ProgramWebviewViewModel`